spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -278,6 +278,10 @@ sealed class SpdxSingleLicenseExpression : SpdxExpression() {
         fun parse(expression: String): SpdxSingleLicenseExpression =
             SpdxExpression.parse(expression) as SpdxSingleLicenseExpression
     }
+    /**
+    * The URL that points to list of spdx licenses on spdx.org
```

Comment:
Missing "." at the end of the sentence.

Reply:
Also "SPDX" should be all caps in free text, and URLs should include a protocol.

Reply:
Fixed